### ISSUE TITLE
Activation time of Gmail Credentials

### ISSUE THREAD

**Issue by antennus** [home-assistant/home-assistant.github.io#29430]
### Feedback

At the end of "Generate Client ID and Client Secret" there should be a note that the activation at Google can take up to 5 hours.

That saves the search for not made mistakes :-))

CU
Andreas 

### URL

https://www.home-assistant.io/integrations/google_mail/

### Version

2023.10.3

**Comment by FletcherAU** [comment]
I stepped through the credential creation process and didn't see any warning from Google that there may be a delay in credential activation. The credentials I created were also usable within a few seconds.

@antennus Do you have a source for the 5 hour window (A google help article etc) so that we can provide an accurate timeframe?

**Comment by antennus** [comment]
@Fletcher
Thank you for your answer.

Unfortunately, I didn't take a screenshot.
This was my procedure:
I went through the activation process as described and completed it without errors.
Unfortunately, gmail did not work as expected.
So I wanted to restart the activation process. On the first screen I got a message similar to:
"The activation process is still ongoing. It may take up to 5 hours to complete. Be patient."

After about 20 minutes, everything worked fine.
Unfortunately, I cannot say any more.

Maybe it's only PICNIC :-))
